selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/SeleniumWebDriverHelper.java

```diff
@@ -451,33 +453,35 @@ public void moveCursorTo(WebElement webElement) {
   }
 
   /**
-   * Checks visibility state of {@link WebElement} with provided locator {@link By}
+   * Checks visibility state of {@link WebElement} with provided locator {@code By}
    *
    * @param elementLocator
    * @return state of element visibility
    */
-  public boolean isVisibleDuringTimeout(By elementLocator) {
+  public boolean isVisible(By elementLocator) {
     try {
-      webDriverWaitFactory
-          .get(ATTACHING_ELEM_TO_DOM_SEC)
-          .until(visibilityOfElementLocated(elementLocator));
-      return true;
-    } catch (TimeoutException ex) {
+      return seleniumWebDriver.findElement(elementLocator).isDisplayed();
+    } catch (NoSuchElementException ex) {
+      return false;
+    } catch (Exception ex) {
+      LOG.info("Unhandled {} in \"isVisible()\" method", ex.getClass().getSimpleName());
```

Comment:
We can omit `in \"isVisible()\" method` string so as log4j prints reference itself, but logging this info is useless IMHO.

Reply:
If we talking about very rare exception which should be took into account, it's better not to handle another type of exceptions here and let client code to work around the problem.